.github/workflows/nv-flash-attn.yml

```diff
@@ -0,0 +1,53 @@
+name: nv-flash-attn
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - 'deepspeed/sequence/**'
+      - 'tests/unit/sequence_parallelism/**'
+      - '.github/workflows/nv-flash-attn.yml'
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
Should we add a cron trigger as well to this?  And perhaps the Nightly CI failure issue creation from [here](https://github.com/microsoft/DeepSpeed/blob/351569dd4a00dea0e00040a816cbc44b1e38a214/.github/workflows/nv-sd.yml#L67C1-L74C32) - that way if a PR doesn't trigger this we will still know if it fails?

Reply:
Thank you @loadams for the suggestion! I added both. Can you check them?

Reply:
Yes this looks good now!